packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js

```diff
@@ -152,32 +153,22 @@ skipInNode(describe)('plugin-meetings', () => {
       .then(() => testUtils.addMedia(guest))
       .catch((e) => { console.error('Error chris joining the meeting ', e); throw e; }));
 
-    it('alice Leaves the meeting', () =>
-      Promise.all([
-        testUtils.delayedPromise(alice.meeting.leave()),
-        testUtils.waitForEvents([{scope: chris.meeting.members, event: 'members:update'}])
-          .then((response) => {
-            const aliceParticipant = response[0].result.delta.updated.find((member) => alice.meeting.members.selfId === member.id);
-
-            assert.equal(aliceParticipant.status, 'NOT_IN_MEETING');
-          })
-      ]).then(() => testUtils.waitForStateChange(alice.meeting, 'LEFT')));
+    it('alice Leaves the meeting', () => Promise.all([
+      testUtils.delayedPromise(alice.meeting.leave()),
+      testUtils.waitForEvents([{scope: chris.meeting.members, event: 'members:update', match: testUtils.checkParticipantUpdatedStatus(alice, 'NOT_IN_MEETING')}])
+    ]).then(() => testUtils.waitForStateChange(alice.meeting, 'LEFT')));
 
-    it('bob and chris leave meeting', () => Promise.all([
-      testUtils.delayedPromise(bob.meeting.leave()),
-      testUtils.waitForEvents([{scope: chris.meeting.members, event: 'members:update'}])
-        .then((response) => {
-          const bobParticipant = response[0].result.delta.updated.find((member) => bob.meeting.members.selfId === member.id);
+    it('bob and chris leave meeting', () =>
+      Promise.all([
+        testUtils.delayedPromise(bob.meeting.leave()),
+        testUtils.waitForEvents([{scope: chris.meeting.members, event: 'members:update', match: testUtils.checkParticipantUpdatedStatus(bob, 'NOT_IN_MEETING')}])
+      ]).then(() => testUtils.waitForStateChange(bob.meeting, 'LEFT'))
+        .then(() => {
+          console.log('CHRIS MEETING', chris.meeting.locusInfo);
```

Comment:
Is this console.log necessary?

Reply:
it was there before..., I guess someone put it there thinking it could be useful when looking at logs of a failed test, but actually these logs are not shown in the log by default, so I've removed it now.